contrib/completions/bash/openshift

```diff
@@ -320,6 +320,7 @@ _openshift_start_kubernetes_apiserver()
     flags+=("--kubelet-https")
     flags+=("--kubelet-port=")
     flags+=("--kubelet-timeout=")
+    flags+=("--kubernetes-service-node-port=")
```

Comment:
This needs a separate commit or it will slip in the next rebase.  In this particular case it doesn't really matter since its a generated file, but you should update for hygiene.